pkg/parca/parca.go

```diff
@@ -74,7 +75,7 @@ type Flags struct {
 }
 
 // Run the parca server
-func Run(ctx context.Context, logger log.Logger, reg *prometheus.Registry, flags *Flags) error {
+func Run(ctx context.Context, logger log.Logger, reg *prometheus.Registry, flags *Flags, uiData map[string]string) error {
```

Comment:
Any reason why we wouldn't make `uiData` a struct?

Reply:
agree, let's go with type safety -> changed